app/queries/facility_yearly_follow_ups_and_registrations_query.rb

```diff
@@ -22,11 +22,11 @@ def call
     records_per_period = records.group_by { |facility|
       year(facility.month_date)
     }
-    records_per_period.each_with_object(Hash.new(0)) do |record_per_period, result|
+    records_per_period.each_with_object([]) do |record_per_period, result|
```

Comment:
Looks like this can just be a `records_per_period.map` now!

Reply:
yeah, that's right